### PR TITLE
validation: check all contracts for correct superchainConfig()

### PR DESCRIPTION
**Pull request by bitwiseguy**
Updates the `testSuperchainConfig` to ensure that the following contracts are all pointed at the standard superchain-wide `SuperchainConfig` contract.
```
* AnchorStateRegistry
* DelayedWETH
* L1CrossDomainMessenger
* L1ERC721Bridge
* L1StandardBridge
* OptimismPortal
```
Previously we were only checking the OptimismPortal contract

### Tests
I manually ran the following to ensure our existing standard chains pass this test:
```
just validate 10 // op-mainnet
just validate 11155420 // op-sepolia
```

### CODE REVIEW

File: add-chain/main.go
Comment (geoknee) [inline add-chain/main.go:93]: Were the changes to this file intentional?

File: validation/superchain-config_test.go
Comment (geoknee) [inline validation/superchain-config_test.go:28]: Nit: Do you think it would be worth enshrining more of this information into the standard config toml files? I think there is precedent for declaring that `contractFoo.MethodBar` should resolve to `AddressBaz` already.

**Review by geoknee — APPROVED** [review]
Approving but we should hold off merging until the governance question resolves.

File: add-chain/main.go
Comment (bitwiseguy) [reply add-chain/main.go:93]: Good catch. They are intentional but I will separate those changes into a different pr to keep this one cleaner

File: validation/superchain-config_test.go
Comment (bitwiseguy) [reply validation/superchain-config_test.go:28]: Since changing those toml files requires a governance approval, I think we should hold off on enshrining this change for now

**Pull request merged**
